templates/catalog/_partials/product-add-to-cart.tpl

```diff
@@ -45,8 +45,8 @@
           {include file='components/qty-input.tpl'
             attributes=[
               "id"=>"quantity_wanted",
-              "value"=>"{if $product.quantity_wanted}{$product.quantity_wanted}{else}1{/if}",
-              "min"=>"{if $product.quantity_wanted}{$product.minimal_quantity}{else}1{/if}"
+              "value"=>"1",
+              "data-min"=>"{$product.minimal_quantity}"
```

Comment:
Please change `=>` to ` => ` 😉

Why not use `min` attribute directly?

Reply:
Oh yeah, that's a good remark!
I will change that to use the `min` attribute directly.